contracts/truefi2/mocks/MockUsdc.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockUsdc is ERC20 {
```

Comment:
We already have a TestUSDCToken deployed to Ropsten in `contracts/trusttoken/mocks/`. Can we reuse that code here, or do you think this mock should be maintained separately?

Reply:
Sure I can use that one. I didn't really realise it is there